packages/experimental/Button/src/ButtonColorTokens.macos.ts

```diff
@@ -0,0 +1,95 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { ButtonTokens } from './Button.types';
+
+export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t: Theme) =>
+  ({
+    backgroundColor: t.colors.defaultBackground,
```

Comment:
Not blocking/doesn't need to be addressed soon, but we should eventually move away from using the old semantic colors and hooking into alias tokens